src/scratch/lag_ditt_eget_spill/README.md

```diff
@@ -1,38 +1,62 @@
+- [ ] __Programmering, 10. trinn__: overføre løsninger til nye problemer ved å generalisere og tilpasse eksisterende programkode og algoritmer.
+- [ ] __Teknologi i praksis, 10. trinn__: vurdere teknologiske produkt ut frå brukartilpassing, HMS-krav og miljøtilpassing
+- [ ] __Teknologi i praksis, 10. trinn__: utvikle ein realistisk kravspesifikasjon for eit teknologisk produkt og beskrive kva behov produktet skal dekkje
+
+## Forslag til læringsmål {.challenge}
+- [ ] Elevene kan samarbeide med andre om ideer og design i arbeidsprosessen.
+- [ ] Elevene kan bruke enkle funksjoner i digitale bildebehandlingsprogram for å designe formelementer til spill.
+- [ ] Elevene kan lage og sette inn lydopptak.
+- [ ] Elevene kan lage et spill med en gjennomtenkt historie, som kombinerer ord, lyd og bilde.
+- [ ] Elevene kan lage hensiktsmessige delproblemer som kan løses med kjente strategier, og sette sammen delproblemene igjen.
+- [ ] Elevene kan dokumentere sin egen kode med kommentarer, og forklare den for en medelev eller klassen.
+- [ ] Elevene kan bruke grunnleggende prinsipper i programmering for å lage et spill.
+- [ ] Elevene kan lese og videreutvikle tidligere utformet kode til å passe i sitt eget spill.
+- [ ] Elevene kan vurdere spillets brukervennlighet, og iverksette nødvendige endringer.
+- [ ] Elevene kan lage en realistisk kravspesifikasjon for spillet sitt, og finne ut hvilke elementer som må være med for at det skal fungere som ønsket.
+
+## Forslag til vurderingskriterier {.challenge}
+- [ ] Eleven viser middels måloppnåelse ved å fullføre oppgaven.
+- [ ] Eleven viser høy måloppnåelse ved å videreutvikle egen kode basert på oppgaven.
+
+## Forutsetninger og utstyr {.challenge}
+- [ ] __Forutsetninger__: Elevene må ha god kunnskap i Scratch. De bør ha gjort flere prosjekter på erfaren-nivået før de begynner med denne oppgaven. Elevene skal designe sine egne ressurser til spillet, inkludert figurer, bakgrunner og lyder. Til dette trenger de papir og noe å skrive med. Ha gjerne klart ekstra papir slik at de kan prøve ut mange forskjellige ideer. Husk at design og planlegging kan ta både én og to dobbelttimer - sørg for at elevene disponerer tiden godt!
+- [ ] __Utstyr__: Datamaskiner med Scratch installert. Eventuelt kan elevene bruke Scratch i nettleseren dersom de har en bruker (eller registrerer seg) på [scratch.mit.edu/](http://scratch.mit.edu/){target=_blank}.
+
+# Fremgangsmåte
+[Klikk her for å se oppgaveteksten.](../lag_ditt_eget_spill/lag_ditt_eget_spill.html){target=_blank}
+_Vi har dessverre ikke noen tips, erfaringer eller utfordringer tilknyttet denne oppgaven enda._
+
+## Variasjoner {.challenge}
+- [ ] Oppgaven har så mange variasjoner elevene ønsker. Se andre prosjekter for inspirasjon.
+
+## Eksterne ressurser {.challenge}
+- [ ] Foreløpig ingen eksterne ressurser...
```

Comment:
i teknologibransjen er det mange som går gjennom en Google Design Sprint når de skal utforske ideer og prototype produkter, og eg tror det er mange gode tips her for de som ønsker å gjøre denne oppgaven skikkelig. eg ville linket til et [foredrag av Braden Kowitz på webdagene 2016](http://2016.webdagene.no/foredrag/google-design-sprint) og [den offisielle siden til Google Design Sprint](http://www.gv.com/sprint/). begge ressursene er på engelsk, men er veldig gode